recipe/meta.yaml

```diff
@@ -48,7 +43,7 @@ test:
     - pip check
```

Comment:
Should not it be 'pytest', not 'py.test'?

Reply:
> Should not it be 'pytest', not 'py.test'?

Yes! Good eye! Made those changes